core/app/modules/kit-library/data/base-controller.php

```diff
@@ -20,9 +22,18 @@ abstract class Base_Controller extends Controller {
 	 */
 	public function get_repository() {
 		if ( ! $this->repository ) {
+			/** @var Module $connect */
```

Comment:
Maybe use the whole namespace since `Module` is generic?

```suggestion
			/** @var Elementor\Core\Common\Modules\Connect\Module $connect */
```

Reply:
DONE